flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/EICAS/ECAM/A320_Neo_UpperECAM.js

```diff
@@ -491,6 +491,48 @@ var A320_Neo_UpperECAM;
                             },
                         ]
                     },
+                    {
+                        name: "L/G",
+                        messages: [
+                            {
+                                message: "GEAR NOT DOWN",
+                                level: 3,
+                                page: "WHEEL",
+                                isActive: () => (
+                                    this.getCachedSimVar("L:A32NX_LDG_NOT_DOWN", "Bool")
+                                )
+                            },
+                        ]
+                    },
+                    {
+                        name: "AUTO FLT",
+                        messages: [
+                            {
+                                message: "AP OFF",
+                                level: 3,
+                                warningoff: true,
```

Comment:
**question**: what does `warningoff` (wrong casing) do?

Reply:
In ECAM, if there is a warning or caution, always play warning sound.
you can check it under "this.hasWarnings = true;"

so that warningoff bypass warning sound

Reply:
Looking at the first image in your PR description, it seems in this situation the audio also plays.

Reply:
no that sound is cavalry charge sound. not a CRC sound.
so must need that warningOff

Reply:
`warningOff` is too undescriptive. I still don't know what it actually does.